docs/contributor/03-10-hyperscaler-account-pool.md

```diff
@@ -82,3 +83,155 @@ metadata:
     tenant-name: {TENANT_NAME}
     hyperscaler-type: "gcp_cf-sa30"
 ```
+
+## Selection Rules
+
+### Overview
+
+HAP evaluates a set of rules to determine what labels to use when querying secret bindings. Input to the rules consists of an SKR's plan, hyperscaler and region. By default, only unchanged hyperscaler type value (values like aws, azure etc.) is used to search for the right pool.  If evaluated to true the rules modify or add labels used in the secret bindings resource query. There are four possible rules to configure:
+* `hap.platformRegionRule` - if evaluated to true the `_<PLATFOR_REGION>` is appended to the `hyperscaler-type` label when searching, refered to as platform region based search. 
+* `hap.clusterRegionRule` - if evaluated to true the `_<HYPERSCALER_REGION>` is appended to the `hyperscaler-type` label when searching, refered to as cluster region based search.
```

Comment:
Would it be convenient to specify what if two types of region are in play?
From BNF notation I know how it is supposed to look like, but nevertheless example could be easier to understand.

Reply:
Let's discuss this modification together.